q2_demux/plugin_setup.py

```diff
@@ -125,7 +125,10 @@
     function=q2_demux.summarize,
     inputs={'data':
             SampleData[SequencesWithQuality | PairedEndSequencesWithQuality]},
-    parameters={'n': qiime2.plugin.Int},
+    parameters={
+      'n': qiime2.plugin.Int,
+      'delta': qiime2.plugin.Int
```

Comment:
Ok.  How does that effect the way I wright the params in the function itself then?  Eg,

````PYTHON
def summarize(output_dir: str, data: _PlotQualView,
              n: int=10000, delta: int % Range(0, None)) -> None:
````
Something like that?